helm/defectdojo/values.yaml

```diff
@@ -124,6 +124,7 @@ celery:
       limits:
         cpu: 2000m
         memory: 256Mi
+    revisionHistoryLimit: 1
```

Comment:
Maybe we should keep default k8s setting as is 10, and let user to specify less

Reply:
Done. I put it in values as opposed to using default in the template or excluding it entirely from the template. I thought it'd be more appropriate to be specific about the value as opposed to letting the API load said default or using the [default directive](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/) from helm. But if there is a strong opinion one way or another, I can change it!

Reply:
The thing is if some of the users are relaying on history feature (10) , it will break their understanding of their current setup